packages/stores/types.ts

```diff
@@ -937,12 +939,66 @@ type TTradersHubStore = {
 
 type TContractReplay = {
     contract_store: {
+        accumulator_previous_spot_time: number | null;
+        barriers_array: Array<unknown> | [];
```

Comment:
In console `barriers_array` was `[ChartBarrierStore]`, but I'm not sure if I should import `ChartBarrierStore` and use it here. Please, fill free to share you thoughts.

Reply:
export type TCoreStores = {
    client: TClientStore;
    common: TCommonStore;
    ui: TUiStore;
    portfolio: TPortfolioStore;
    contract_trade: TContractTradeStore;
    // This should be `any` as this property will be handled in each package.
    // eslint-disable-next-line @typescript-eslint/no-explicit-any
    modules: Record<string, any>;
    notifications: TNotificationStore;
    traders_hub: TTradersHubStore;
    gtm: TGtmStore;
    pushwoosh: Record<string, unknown>;
    contract_replay: TContractReplay;
    chart_barrier_store: Record<string, unknown>;
    active_symbols: TActiveSymbolsStore;
};

chart_barrier_store already has a place holder so you can probably make reference to it like `TCoreStore['chart_barrier_store']`.

And it'll be typed or TS converted at some point i guess.

Reply:
Thank you, will apply